### PR TITLE
Fixed typos in TFLite metadata_writer_tutorial.ipynb

### PR DESCRIPTION
**Pull request by LakshmiKalaKadali**
Several typos are fixed in TFLite metadata_writer_tutorial.ipynb

### CODE REVIEW

File: tensorflow/lite/g3doc/models/convert/metadata_writer_tutorial.ipynb [hunk 1]
```diff
@@ -223,7 +223,8 @@
         "    writer_utils.load_file(_MODEL_PATH), [_INPUT_NORM_MEAN], [_INPUT_NORM_STD],\n",
         "    [_LABEL_FILE])\n",
         "\n",
-        "# Verify the metadata generated by metadata writer.\n",
+        "# Verify the metadata generated by the 
+metadata writer.\n",
```
Comment (talumbau) [inline tensorflow/lite/g3doc/models/convert/metadata_writer_tutorial.ipynb:227]: It looks like there is an inadvertent new line inserted here, perhaps?

**Review by talumbau — COMMENTED** [review]
Just a comment on a single line that seemed off. Rest looks good. Thanks very much!

File: tensorflow/lite/g3doc/models/convert/metadata_writer_tutorial.ipynb (hunk 1 above)
Comment (LakshmiKalaKadali) [reply tensorflow/lite/g3doc/models/convert/metadata_writer_tutorial.ipynb]: Suggested changes are done. Thank You

**Comment by gbaned** [comment]
Hi @talumbau Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @talumbau Can you please review this PR ? Thank you!

**Comment by keerthanakadiri** [comment]
Hi @talumbau Can you please review this PR ? Thank you!

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**